flags/eigendaflags/deprecated.go

```diff
@@ -28,8 +28,8 @@ func withDeprecatedFlagPrefix(s string) string {
 	return "eigenda-" + s
 }
 
-func withDeprecatedEnvPrefix(envPrefix, s string) string {
-	return envPrefix + "_" + s
+func withDeprecatedEnvPrefix(envPrefix, s string) []string {
+	return []string{envPrefix + "_" + s}
```

Comment:
The reason why we had changed this is because returning an array like this prints an ugly msg for deprecated flags. It says something like "flag --routing.xyz (env var [ROUTING.XYZ]) is deprecated"

Reply:
Gotcha - will update

Reply:
[3a6f3fb](https://github.com/Layr-Labs/eigenda-proxy/pull/195/commits/3a6f3fbd9f7ce92bbfe3b98871724bc7b3829a94)